### DISCUSSION TITLE
Cannot get merchant to not auto-check every few seconds

### DISCUSSION THREAD

**Discussion by aretelew** [jef/streetmerchant#1709] [Help]
Hello everyone. I can't get the mechant to not auto-check every few seconds. I'm not really worried about bestbuy and amazon, but I've seen many people get IP banned from Newegg from excessive checking. I've set the page_sleep_min to 60000, or 1 minute, but nothing changes. I tried to set the time in the stores as so: amazon, bestbuy, newegg:60000:90000, but that didn't work either. What should I do?

**Comment by TheRealce** [comment]
You can bypass this ban by changing your IP. You can do this by getting a free VPN and changing your server your connected to or use this video: "https://www.youtube.com/watch?v=z7Mm9YqeaEY&ab_channel=PrestonHudman" to MAC clone to change your IP. You only need to do this at the beginning of each day using the bot.
